api-specificatie/openapi.yaml

```diff
@@ -1163,28 +1156,27 @@ components:
             $ref: '#/components/schemas/AdresseerbaarObject_links'
     AdresseerbaarObjectMogelijkOnjuist:
       type: object
+      description: Wanneer hier een property is opgenomen met de waarde true, dan is de waarde van het gelijknamige property in de resource mogelijk onjuist. De juistheid van dit gegeven wordt op dit moment onderzocht. Onder property toelichting wordt toegelicht wat er mogelijk onjuist is aan het betreffende gegeven.
       properties:
-        gebruiksdoelen:
-          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+        gebruiksdoel:
```

Comment:
Hier heb je enkelvoud gebruikt terwijl dat in de resource meervoud is. lijkt mij dat je dezelfde property-naam hanteert.

Reply:
dank je, het is nu weer gebruiksdoelen